kratos/includes/geometrical_object.h

```diff
@@ -74,21 +75,27 @@ class GeometricalObject : public IndexedObject
     ///@{
 
     /// Default constructor.
-    explicit GeometricalObject(IndexType NewId = 0) : BaseType(NewId),
-        mpGeometry()
+    explicit GeometricalObject(IndexType NewId = 0)
+        : BaseType(NewId),
```

Comment:
I would suggest removing the `typedef IndexedObject BaseType` and replace `BaseType` with `IndexedObject` for clarity.